common/metrics/metric_defs.go

```diff
@@ -465,6 +474,8 @@ const (
 	OperationArchivalQueueProcessorScope = "ArchivalQueueProcessor"
 	// OperationMemoryScheduledQueueProcessorScope is a scope for memory scheduled queue processor.
 	OperationMemoryScheduledQueueProcessorScope = "MemoryScheduledQueueProcessor"
+	// OperationCallbackQueueProcessorScope is a scope for the callback queue processor.
+	OperationCallbackQueueProcessorScope = "CallbackQueueProcessor"
```

Comment:
Looks like we have some duplicated definition here...

Reply:
Yup, same for transfer tasks and timer tasks BTW.

I just duplicated what we had for transfer tasks.

Reply:
yeah.. I will clean them up.